ebike_city_tools/metrics.py

```diff
@@ -47,7 +47,7 @@ def od_sp(G, od, weight, weight_od_flow=False):
         if row["trips"] == 0:
             continue
         # compute shortest path
-        sp_len = nx.shortest_path_length(G, source=row["s"], target=row["t"], weight=weight)
+        sp_len = nx.shortest_path_length(G, source=row["source"], target=row["target"], weight=weight)
```

Comment:
This is a breaking change for other parts of the backend. In my evaluation scripts, the attributes are probably called "s" and "t" somewhere, so I can't merge this change. A good solution in these cases is usually to add these names as keyword arguments, like 
```
def od_sp(G, od, weight, weight_od_flow=False, source_col_name="s", target_col_name="t"):
       .....
       sp_len = nx.shortest_path_length(G, source=row[source_col_name], target=row[target_col_name], weight=weight)
```

Reply:
I changed it back to the original naming and adjusted it on our side. So no changes needed in the metrics.py file.